application/views/main/notification.php

```diff
@@ -1,31 +1,10 @@
 <?php
```

Comment:
We can remove this php block completely

Reply:
If I do you have to go through approval again...